<launch>
  <!-- Search demo -->
  <param name="/dijkstra_planner_demos/search_demo" value="true" />
  <node pkg="dijkstra_planner_demos" type="dijkstra_planner_demos" name="search_demo" output="screen">
    <param name="mapImgDirectory" value="$(find dijkstra_planner_demos)/map/maze.png" />
    <param name="gridResolution" value="0.1" />
    <param name="safeDistance" value="0.0" />
    <param name="robotX" value="0.1" />
    <param name="robotY" value="0.1" />
    <param name="goalX" value="30" />
    <param name="goalY" value="2" />

  </node>

  <!-- Rviz -->
  <node pkg="rviz" type="rviz" name="rviz_search_demo"
    args="-d $(find dijkstra_planner_demos)/launch/rviz/search.rviz" />

</launch>